### PR TITLE
High-Class Corporate Luxury Ship

### PR DESCRIPTION
**Pull request by Mannybrado**
## About The Pull Request
Adds in a brand new ship, the High-Class. First(probably) in the new line of Corporate Luxury ships. Includes a boardroom and stuff. Also, a sec-tec vendor.

## Why It's Good For The Game
ceo of shiptest,
more roleplay ships are good i think

## Changelog
:cl:
add: new ship, high-class
/:cl:

### CODE REVIEW

**Comment by ValoTheValo** [comment]
Could you post a picture of the ship?

**Comment by Mannybrado** [comment]
corpship
here. you can also look at the mapdiff bot, but, guh

**Comment by ValoTheValo** [comment]
Oooh very nice

**Comment by Mannybrado** [comment]
buhh trying to fix linters

**Comment by Mannybrado** [comment]
videogames

File: _maps/configs/corphigh.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+	"map_name": "High-class Corporate Luxury Ship",
+	"map_short_name": "High-class",
+	"map_path": "_maps/shuttles/shiptest/corp_high.dmm",
+	"map_id": "corp_high",
+	"job_slots": {
+		"Captain": 1,
+		"Security Officer": 2,
+		"Station Engineer": 1,
```
Comment (Anderson-Cooper) [inline _maps/configs/corphigh.json]:
```suggestion
		"Station Engineer": 2,
```

Comment (Anderson-Cooper) [reply _maps/configs/corphigh.json]: we need more engineers.

**Review by Anderson-Cooper — CHANGES_REQUESTED** [review]

File: _maps/configs/corphigh.json (hunk 1 above)
Comment (Mannybrado) [reply _maps/configs/corphigh.json]: done

**Comment by triplezeta** [comment]
classy indeed :+1:

**Review by triplezeta — APPROVED** [review]
looks solid

**Pull request merged**